plugins/Makefile

```diff
@@ -3,7 +3,7 @@
 ifeq ($(FTE_TARGET),win32)
 	PLUG_NATIVE_EXT=_x86.dll
 	PLUG_LDFLAGS= -static-libgcc
-	PLUG_LDFLAGS_ZLIB=-L../engine/libs/mingw-libs -lzlib
+	PLUG_LDFLAGS_ZLIB=-L../engine/libs/mingw-libs -lz
```

Comment:
Fixes the win32 plugins build, same as the win64 case a few lines down. Think it's correct at least.